osidb/filters.py

```diff
@@ -479,3 +480,25 @@ class Meta:
             "order": ["exact"],
             "external_system_id": ["exact"],
         }
+
+
+class FlawCVSSFilter(FilterSet):
+    class Meta:
+        model = FlawCVSS
+        fields = {
+            "comment": ["exact"],
+            "created_dt": ["exact"]
+            + LT_GT_LOOKUP_EXPRS
+            + LTE_GTE_LOOKUP_EXPRS
+            + DATE_LOOKUP_EXPRS,
+            "flaw": ["exact"],
+            "issuer": ["exact"],
+            "score": ["exact"],
+            "updated_dt": ["exact"]
+            + LT_GT_LOOKUP_EXPRS
+            + LTE_GTE_LOOKUP_EXPRS
+            + DATE_LOOKUP_EXPRS,
+            "uuid": ["exact"],
+            "vector": ["exact"],
+            "version": ["exact"],
```

Comment:
I am unsure whether this line is correct, because `FlawCVSSSerializer` contains `cvss_version = serializers.CharField(source="version")`.  But when using `cvss_version` instead of `version`, I get `TypeError: 'Meta.fields' must not contain non-model field names: cvss_version`, so probably it is.

Reply:
That is quite undesirable because from the user point of view when the attribute is called differently in the filter than elsewhere. There should be some conversion or something. Otherwise the version filter could be useful.